### DISCUSSION TITLE
Dropdown not loading all elements

### DISCUSSION THREAD

**Discussion by adnanawan12** [payloadcms/payload#1356] [Q&A]
Dear,

Dropdown list is not loading all elements on Windows/Mobile Chrome while on Mac OS chrome is working fine. I have 78 items in company collection only few are appearing.

https://user-images.githubusercontent.com/6416296/200482470-044edbde-c5d1-4918-bf3d-7fa7bd44e6ff.mp4

**Comment by jacobsfletch** [comment]
Hey @adnanawan12! I was able to reproduce this on my end (Chrome, Mac OS), and can confirm that dragging the scrollbar does not fire the `onMenuScrollToBottom` method in [react-select](https://react-select.com/home). It looks like this is a known issue, described here https://github.com/JedWatson/react-select/issues/3232. Somebody even opened a PR to fix it: https://github.com/JedWatson/react-select/pull/4970 but that was almost exactly 1 year ago.
